tests/tools_tests/ppx/TestPpx.res

```diff
@@ -61,3 +61,7 @@ let eq2 = 3 === 3
 
 let test = async () => 12
 let f = async () => (await test()) + 1
+
+module Fragments = {
+  let f1 = <> </>
```

Comment:
This should be `<></>`, no? (without the space).

Reply:
Sounds reasonable, although I do believe this is the current behavior. ([Playground](https://rescript-lang.org/try?version=v11.1.4&module=esmodule&code=DYUwLgBAZgjBC8EA8A+ZB6FAoIA))